samples/apps/copilot-chat-app/SKWebApi/Program.cs

```diff
@@ -24,28 +25,42 @@ public static void Main(string[] args)
             serverPort = SKWebApiConstants.DefaultServerPort;
         }
 
+        // Set the protocol to use
         string protocol = "https";
+        bool useHttp = builder.Configuration.GetSection("UseHttp").Get<bool>();
+        if (useHttp)
```

Comment:
this block can be combined with line 29

`string protocol = useHttp ? "http" : "https";`

Reply:
Changed